administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -0,0 +1,41 @@
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `description` text NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `date` DATE NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `published` tinyint NOT NULL DEFAULT 1,
+  `ordering` int NOT NULL DEFAULT 0,
+  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `created_by` int unsigned NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `modified_by` int unsigned NOT NULL DEFAULT 0,
+  `state` TINYINT NOT NULL DEFAULT '1'
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS `#__guidedtours` (
  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `description` text NOT NULL,
  `title` varchar(255) NOT NULL,
  `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `published` tinyint NOT NULL DEFAULT 1,
  `ordering` int NOT NULL DEFAULT 0,
  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `created_by` int unsigned NOT NULL DEFAULT 0,
  `modified` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `modified_by` int unsigned NOT NULL DEFAULT 0,
  `state` TINYINT NOT NULL DEFAULT '1'
```
`CURRENT_TIMESTAMP()` is a function in MySQL and so needs the brackets.

And it is not a valid default for the `date` data type.

That's why you get an error "#1067 - Fehlerhafter Vorgabewert (DEFAULT) für 'date'".

Since we do not use that data type in Joomla but `datetime` also for dates, I suggest to change it to `datetime`.